rd_ui/app/scripts/ng_highchart.js

```diff
@@ -325,6 +325,36 @@
                 series = seriesCopy;
               }
 
+              if (chartOptions['sortY'] === true || chartOptions['sortY'] === undefined) {
+                var sortable = true;
+
+                for (var i = 0; i < series.length; i++) {
+                  if (series[i].data.length != 1) {
```

Comment:
Isn't `series[i].data.length` the array of points in the chart? In that case, why only sort if length is 1? Am I missing something?